const throwError = () => {
  throw new Error(`FS operation failed`);
};

export default throwError;
